### DISCUSSION TITLE
I would like to create a static generation function.

### DISCUSSION THREAD

**Discussion by Yuutakasan** [Laravel-Backpack/community-forum#574] [Ideas]
I am currently trying to use clever to output static, but I have a feeling that Ajax related functions will stop working. I would like to see official support for this.
https://usecleaver.com/

**Comment by tabacitu** [comment]
I'm sorry @Yuutakasan but I don't see how this has anything to do with Backpack. You can use whatever you want for your application side, Backpack only takes cares of the admin panel. It will not conflict with whatever you do on your app code.
